lib/pharos/phases/configure_host.rb

```diff
@@ -22,7 +22,11 @@ class ConfigureHost < Pharos::Phase
 
       def call
         logger.info { "Configuring essential packages ..." }
-        exec_script('configure-essentials.sh')
+        exec_script(
+          'configure-essentials.sh',
+          HTTP_PROXY: @host.http_proxy.to_s,
+          SET_HTTP_PROXY: @host.http_proxy.nil? ? 'false' : 'true'
```

Comment:
This could be handled in the script instead

Reply:
How?